test/e2e/storage/OWNERS

```diff
@@ -9,17 +9,21 @@ approvers:
 - msau42
 - davidz627
 - xing-yang
+- pohly
 reviewers:
-- saad-ali
 - rootfs
 - gnufied
 - jingxu97
 - jsafrane
 - msau42
-- jeffvance
-- copejon
 - verult
 - davidz627
 - xing-yang
+- pohly
 labels:
 - sig/storage
+
+emeritus_reviewers:
```

Comment:
Can you also move rootfs and davidz627 to emeritus as well?

And we can pull in the sig-storage-approvers/reviewers aliases

Reply:
Done. Let's also update that alias list.